pramen-py/src/pramen_py/runner/runner_transformation.py

```diff
@@ -67,16 +68,27 @@ def overwrite_info_dates(
     return config
 
 
-def discover_transformations(
-    path: pathlib.Path = TRANSFORMATIONS_DIR,
-) -> Generator[Type[T_TRANSFORMATION], None, None,]:
+def get_transformations_dir(dir_key: str, envs: Env) -> pathlib.Path:
+    transformations_dir_path = os.environ.get(dir_key) or ""
+    if not transformations_dir_path:
+        transformations_dir_path = envs.str(dir_key, "")
+    return pathlib.Path(
+        transformations_dir_path or DEFAULT_TRANSFORMATIONS_DIR
+    )
+
+
+def discover_transformations() -> Generator[
+    Type[T_TRANSFORMATION], None, None
+]:
     """Return generator of Transformation subclasses.
 
     In order to provide a set of subclasses we keep track already
     provided class names and do not allow to yield a class with the same
     name more than one time.
     """
-    load_modules(path)
+    load_modules(
+        get_transformations_dir("PRAMENPY_TRANSFORMATIONS_NAMESPACE", env)
```

Comment:
You don't need to pass 'PRAMENPY_TRANSFORMATIONS_NAMESPACE' here since it is not expected to change. You can use it hardcoded inside 'get_transformations_dir()' or create a constant for it.